dockerfiles/Dockerfile.training

```diff
@@ -158,6 +158,10 @@ RUN pip install azureml-defaults sentencepiece==0.1.92 transformers==2.11.0 msgp
 FROM nvidia/cuda:10.2-cudnn8-runtime-ubuntu18.04
 WORKDIR /stage
 
+# install curl, git, ssh (required by MPI when running ORT tests)
```

Comment:
copied comment is inaccurate. also, should we run unattended-upgrade for the final stage?